sw/device/lib/dif/dif_clkmgr.c

```diff
@@ -88,8 +88,19 @@ dif_result_t dif_clkmgr_jitter_set_enabled(const dif_clkmgr_t *clkmgr,
     default:
       return kDifBadArg;
   }
-  mmio_region_write32(clkmgr->base_addr, CLKMGR_JITTER_ENABLE_REG_OFFSET,
-                      new_jitter_enable_val);
+
+  multi_bit_bool_t clk_jitter_enable_val =
+      mmio_region_read32(clkmgr->base_addr, CLKMGR_JITTER_ENABLE_REG_OFFSET);
+  if (clk_jitter_enable_val == kMultiBitBool4True &&
+      new_jitter_enable_val != kMultiBitBool4True) {
+    return kDifLocked;
+  }
+
+  if (new_jitter_enable_val == kMultiBitBool4True) {
```

Comment:
This "if" is incorrect: even if we try to write False it will write True. I propose you remove the If, and replace `new_jitter_enable_val` below by `kMultiBitBool4True`.

Reply:
Sorry @matutem , I merged the PR before seeing this. Can we fix this in the follow-up containing also the new software test, please?

Reply:
I made the change in my test PR: https://github.com/lowRISC/opentitan/pull/26058

Reply:
Thanks @matutem !